web/l10n/en.json

```diff
@@ -64,6 +68,7 @@
     "sharedTypeNumber": "Number",
     "sharedTypeBoolean": "Boolean",
     "sharedTimezone": "Timezone",
+    "sharedVolumeUnit": "Volume",
```

Comment:
May be we should add "Unit" word to strings, they look not clearly
![image](https://user-images.githubusercontent.com/5688080/30106829-32d47ff0-9316-11e7-9b36-78002695fb5b.png)

Reply:
Agree.